source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -78,5 +80,7 @@ For each file, create MongoClient initialized with the mongodb+srv connection
 string. You SHOULD verify that the client's initial seed list matches the list of
 seeds. You MUST verify that the set of ServerDescriptions in the client's
 TopologyDescription eventually matches the list of hosts. You MUST verify that
-the set of Connection String Options matches the client's parsed set. You MUST
-verify that an error has been thrown if ``error`` is present.
+echo of the values of the Connection String Options under ``options`` match the
```

Comment:
echo?  not sure what this means.

Reply:
I'm pretty sure this means "print" or something equivalent to allow a comparison check.

Reply:
How about just:

    You MUST verify that the values of the connection string options under ``options`` match the client's parsed values for those options.

Reply:
I meant "each".

Reply:
Fixed too.